samples/unit-tests/series-column/data-decimalpoint-negative/demo.js

```diff
@@ -0,0 +1,26 @@
+QUnit.test("Data module with decimapPoint and negative numbers (#4749)", function (assert) {
+    var chart = $("#container").highcharts({
+        data: {
+            table: 'datatable',
```

Comment:
This test fails because the element id `datatable` is not present in the karma template. The fix would be to insert the data table HTML dynamically into the DOM from inside the test, something like this: 
```
document.body.innerHTML += `<table id="datatable">
    <thead>
        <tr>
            <th></th>
            <th>Jane</th>
            <th>John</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <th>Apples</th>
            <td>-3,4</td>
            <td>-4,24</td>
        </tr>
        <tr>
            <th>Pears</th>
            <td>-1,2</td>
            <td>-1,5</td>
        </tr>
        <tr>
            <th>Plums</th>
            <td>5,1</td>
            <td>11,1</td>
        </tr>
        <tr>
            <th>Bananas</th>
            <td>-1,1</td>
            <td>-1,1</td>
        </tr>
        <tr>
            <th>Oranges</th>
            <td>-3,12</td>
            <td>-2,9</td>
        </tr>
    </tbody>
</table>`;
````